src/Modals/Modal.tsx

```diff
@@ -100,6 +106,8 @@ export interface ModalProps {
    * e.g. initialize values in a form
    */
   onOpen?: () => void;
+  /** Specifies the ref of the Modal */
+  forwardRef?: RefObject<HTMLDivElement>;
```

Comment:
Reading the ref in react they did not emphasize that ref have to be to a div unless its our convention.

Reply:
Jonathan advised in his other PR that 'Refs are generic so we need to provide a specific HTMLElement type to
use with it,' so I included `HTMLDivElement,` since for accessibility purposes the modal header (a div) should be focused for the screen reader.

Reply:
make sense thanks